fetch-submodules.sh

```diff
@@ -75,3 +75,5 @@ fetch_git "https://github.com/libretro/common-overlays.git" "media/overlays"
 fetch_git "https://github.com/libretro/retroarch-assets.git" "media/assets"
 fetch_git "https://github.com/libretro/retroarch-joypad-autoconfig.git" "media/autoconfig"
 fetch_git "https://github.com/libretro/libretro-database.git" "media/libretrodb"
+
+git submodule update --init
```

Comment:
Adding `--recursive` at the end will bring in any third-party dependencies as well. May be a good addition.